lib/config/config-ops.js

```diff
@@ -30,28 +30,28 @@ var RULE_SEVERITY_STRINGS = ["off", "warn", "error"],
 // Public Interface
 //------------------------------------------------------------------------------
 
-module.exports = {
+module.exports = class ConfigOps {
```

Comment:
I think that modules are good namespaces in ES6. Anyhow, you can still use the "class-style" function declaration in objects, maybe something to consider.

e.g.
```js
module.exports = {
  createEmptyConfig () {
    return // ...
  }
}
```